resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/resources/File.java

```diff
@@ -371,6 +391,32 @@ public void setContents(InputStream content, int updateFlags, IProgressMonitor m
 			subMonitor.done();
 		}
 	}
+	@Override
+	public void setContents(byte[] content, int updateFlags, IProgressMonitor monitor) throws CoreException {
+		String message = NLS.bind(Messages.resources_settingContents, getFullPath());
```

Comment:
Inline?

Reply:
I prefer short lines and consistency with the other code around.